pages/createOrder.js

```diff
@@ -163,6 +167,8 @@ class CreateOrderPage extends React.Component {
       return this.renderMessage('warning', intl.formatMessage(messages.missingTier), true);
     } else if (data.Tier && data.Tier.endsAt && new Date(data.Tier.endsAt) < new Date()) {
       return this.renderMessage('warning', intl.formatMessage(messages.expiredTier), true);
+    } else if (data.Collective.settings.disableCustomContributions) {
```

Comment:
**[:no_entry_sign: **Blocker**]** This will prevent **all** contributions (even the ones with tiers). We just want to prevent custom contributions.

```suggestion
    } else if (data.Collective.settings.disableCustomContributions && !data.Tier) {
```

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
Done